### PR TITLE
feat(ui): Improve timezone popover

### PR DESCRIPTION
**Pull request by webjunkie**
## Problem

I always have a hard time reading what is actually important in the timezone popover. The timezone labels are more jarring than useful.

Old:

image

New:

image

## Changes

Adjust so that there is a clear hierarchy of information importance.

### CODE REVIEW

**Review by skoob13 — APPROVED** [review]

File: frontend/src/lib/components/TZLabel/index.tsx
Comment (Twixes) [inline frontend/src/lib/components/TZLabel/index.tsx:1]: Visual hierarchy is great! But in this case it's somewhat chaotic: three different icons having three different sizes is too much variance, and alignment is out of whack.
346566419-2ff00444-b2ff-48fd-a800-3dad44a946e0
While we're at it, it'd also be great to center the popover title vertically in its area.

**Review by Twixes — CHANGES_REQUESTED** [review]

File: frontend/src/lib/components/TZLabel/index.tsx
Comment (webjunkie) [reply frontend/src/lib/components/TZLabel/index.tsx:1]: Agreed... I'll try to enhance this more.

Comment (Twixes) [reply frontend/src/lib/components/TZLabel/index.tsx:1]: Hm, the wonkiness with margins is still there:
Screenshot 2024-07-09 at 18 13 41

Comment (webjunkie) [reply frontend/src/lib/components/TZLabel/index.tsx:1]: Ah, I see. Fixed.

**Review by Twixes — APPROVED** [review]

**Pull request merged**